Lib/pdb.py

```diff
@@ -440,7 +440,7 @@ def displayhook(self, obj):
             self.message(repr(obj))
 
     def default(self, line):
-        if line[:1] == '!': line = line[1:]
+        if line[:1] == '!': line = line[1:].lstrip()
```

Comment:
It's not a big deal, but do we need to specifically do `lstrip` instead of `strip`? It would probably always make the future developer to wonder why we did an `lstrip` here.

Reply:
No obligation to use `lstrip()` to `strip()` here, and I agree about legibility. Changed in 3968695